draft-irtf-cfrg-vdaf.md

```diff
@@ -5098,10 +5098,41 @@ possible. See {{xof-vs-ro}} for more information.
 | FieldLeaf  | `Field255` ({{fields}}) |
 {: #idpf-bbcggi21-param title="Constants and type definitions for the IDPF of BBCGGI21."}
 
-### Key Generation
+### Overview
+
+At a high level, the IDPF maps a key generator's input `(alpha, beta_1, ..., beta_BITS)` onto
+a binary tree with `BITS+1` levels, where each edge going from a parent node to a left child is
+labeled `0`, and each right edge is labeled `1`. Then each leaf node corresponds to a bit string
+of length `BITS`, where the labels on the path from the root to `x` contain the individual bits.
+Finally, all nodes on the tree have have an assigned value, with the nodes on the path from the
+root to `alpha` having values `beta_1, ..., beta_BITS`, and all other nodes having value `0`.
+
+The IDPF construction now boils down to secret-sharing the node labels of that tree in an efficient way. Note
+that explicitly representing the tree requires `O(2^BITS)` space, so the generator cannot just
+compute additive shares of it and send them to the two evaluators. Instead, the evaluators will
+re-generate shares of the tree using an extendable output function (XOF). The basic observation is that if
+both evaluators have the same seed `s` of length `KEY_SIZE`, then expanding `s` using a XOF will also
+result in the same expansion. If we set the length of the XOF expansion to `2*KEY_SIZE`, it can then be
+split again into two seeds `s_l`, `s_r`, that can again serve as XOF seeds. Now if we view seeds as XOR-shares
+of integers, and if evaluators have the same seed at the root of the tree, then their expanded trees will form
+a secret-shared tree of zeros. The actual construction will use a `convert` function that maps seeds
+into the appropriate output domain.
+
+The open task now is to ensure that evaluators have *different* seeds at nodes that lie on the path
+to `alpha`, while having the same seeds on all other nodes. This is done using so-called
+*correction words* that are conditionally added to the XOF output by both evaluators.
```

Comment:
We don't get to use bold. To denote a new term we're introducing, we've been using quotes.
```suggestion
"correction words" that are conditionally added to the XOF output by both evaluators.
```